x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -316,6 +316,7 @@ export const PolicyTabs = React.memo(() => {
                   getArtifactPath={getTrustedAppsListPath}
                   getPolicyArtifactsPath={getPolicyDetailsArtifactsListPath}
                   canWriteArtifact={canWriteTrustedApplications}
+                  CardDecorator={ProcessDescendantsIndicator}
```

Comment:
oh, in this file we need to pass the specific `TrustedAppCardDecorator` and `EventFiltersCardDecorator` components.

(this is something that would be found by type check if `ProcessDescendantsIndicator` requires a non-optional `labels` prop)